### PR TITLE
[added] description for accountz endpoint

### PR DESCRIPTION
**Pull request by matthiashanel**
Signed-off-by: Matthias Hanel <mh@synadia.com>

### CODE REVIEW

File: nats-server/configuration/monitoring.md [hunk 1]
```diff
@@ -520,6 +521,111 @@ The `/subsz` endpoint reports detailed information about the current subscriptio
...
+| Result | Return Code |
+| :--- | :--- |
+| Success | 200 \(OK\) |
+| Error | 400 \(Bad Request\) |
+
+#### Arguments
+
+| Argument | Values | Description |
+| :--- | :--- | :--- |
+| acc | account name | Include detail information for the requested account. Default is empty. When not set, a list of all accounts is included. |
```
Comment (ColinSullivan1) [inline nats-server/configuration/monitoring.md:540]: Might suggest `Include metrics for the specified account`

File: nats-server/configuration/monitoring.md [hunk 2]
```diff
@@ -520,6 +521,111 @@ The `/subsz` endpoint reports detailed information about the current subscriptio
...
+#### Example
+
+* Get list of all accounts:  [http://demo.nats.io:8222/accountz](http://demo.nats.io:8222/accountz)
+* Get details for specific account `$G`:  [http://demo.nats.io:8222/accountz?acc=\$G](http://demo.nats.io:8222/accountz?acc=$G)
+
+#### Response
+
+Default behavior:
+
+```javascript
```
Comment (ColinSullivan1) [inline nats-server/configuration/monitoring.md:551]: json?

**Review by ColinSullivan1 — APPROVED** [review]
One suggestion...  LGTM - Thanks!

**Review by gcolliso — COMMENTED** [review]
lgtm - @matthiashanel is this something to be merged now or wait for 2.2 release?

File: nats-server/configuration/monitoring.md (hunk 1 above)
Comment (matthiashanel) [reply nats-server/configuration/monitoring.md]: incorporated, thanks

**Comment by matthiashanel** [comment]
> lgtm - @matthiashanel is this something to be merged now or wait for 2.2 release?

I think @ColinSullivan1 is going to coordinate that with you

**Pull request merged**